gulpfile.js

```diff
@@ -485,6 +513,19 @@ gulp.task('regenerate:expected:csazurecomposite', function (cb) {
   }, cb);
 });
 
+
+gulp.task('regenerate:expected:go', function(cb){
+  regenExpected({
+    'outputBaseDir': 'src/generator/AutoRest.Go.Tests',
+    'inputBaseDir': 'src/generator/AutoRest.Go.Tests',
+    'mappings': goMappings,
+    'outputDir': 'src/tests/generated',
+    'codeGenerator': 'Go',
+    'flatteningThreshold': '1'
```

Comment:
@vishrut What is ```flatteningThreshold```? We can remove it if not implemented. I know its one of argument of Autorest command but we never use it code generator.